README.md

```diff
@@ -17,440 +14,20 @@ Platform support:
-	| Chip                      | MaterialFilledInputChipStyle<br>MaterialFilledChoiceChipStyle<br>MaterialFilledFilterChipStyle<br>MaterialFilledActionChipStyle<br>MaterialOutlinedInputChipStyle<br>MaterialOutlinedChoiceChipStyle<br>MaterialOutlinedFilterChipStyle<br>MaterialOutlinedActionChipStyle |
-	| ChipGroup                 | MaterialFilledInputChipGroupStyle<br>MaterialFilledChoiceChipGroupStyle<br>MaterialFilledFilterChipGroupStyle<br>MaterialFilledActionChipGroupStyle<br>MaterialOutlinedInputChipGroupStyle<br>MaterialOutlinedChoiceChipGroupStyle<br>MaterialOutlinedFilterChipGroupStyle<br>MaterialOutlinedActionChipGroupStyle |
-	| Divider                   | MaterialDividerStyle |
-
-- Some controls have been removed
-
-	List of the controls and styles that have been removed from Uno.Themes:
-
-	| **Controls**              | **StyleNames**                                                                |
-	|---------------------------|------------------------------------------------------------------------------|
-	| BottomNavigationBar       | MaterialBottomNavigationBarStyle                                              |
-	| ExpandingBottomSheet      | MaterialExpandingBottomSheetStyle                                             |
-	| ModalStandardBottomSheet  | MaterialModalStandardBottomSheetStyle                                         |
-	| StandardBottomSheet       | MaterialStandardBottomSheetStyle                                              |
-	| SnackBar                  | MaterialSnackBarStyle                                                         |
-
-	- BottomNavigationBar was replaced by TabBar in [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui), but it is not an exact 1:1 replacement.
-	In the mean time, if you really need the badge and/or other customizability, two options are available:
-
-		Import locally the old sources (control + style) from Uno.Themes;
-
-		OR
-
-		Copy the MaterialBottomTabBarItemStyle from [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui), and modify the style to suit your needs.
-		(Note that there are two copies of the style, one for iOS and Android and one for rest of the platforms: UWP, Skia, WASM, etc...);
-
-	- For StandardBottomSheet and ModalStandardBottomSheet
-	It's replaced by using a Flyout and the MaterialFlyoutPresenterStyle that is allowing you to have a bottom sheet.
-
-		For example:
-		```xml
-		<Flyout Placement="Full"
-                LightDismissOverlayMode="On"
-                FlyoutPresenterStyle="{StaticResource MaterialFlyoutPresenterStyle}">
-			<Grid MaxHeight="370"
-				  VerticalAlignment="Bottom">
-				...Your bottom sheet content...
-			</Grid>
-		</Flyout>
-		```
-
-	- For SnackBar: so far no replacement for SnackBar has been added to [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui), but it's planned to add one in a future version.
 
```

Comment:
I think that Before The Documentation section we could have a quick "Get Started" section with a quick one-liner and the actual direct link to the get started doc.
Similar to what we have in Uno here https://github.com/unoplatform/uno#getting-started
That way it's easier to click on it when you arrive on the repo and just want to start ;)

Reply:
```suggestion

## Getting Started

See the complete [Getting Started](doc/getting-started.md) guides for starting with this library.

For a larger example and features demo, visit the [Uno Gallery](https://github.com/unoplatform/uno.gallery) repository.

```